miden-lib/asm/sat/prologue.masm

```diff
@@ -362,8 +369,8 @@ end
 #!  - The account data is invalid.
 #!  - Any of the consumed notes do note exist in the note db.
 #!
-#! Stack:       [BH, aid, IAH, NC]
-#! Advice Tape: [aid, ZERO, ZERO, nonce, AVR, ASR, ACR,
+#! Stack:       [BH, acct_id, IAH, NC]
+#! Advice Tape: [acct_id, ZERO, ZERO, nonce, AVR, ASR, ACR,
```

Comment:
nit: we'll need to rename `advice tape` into `advice stack` here and in other places.